src/scripts/main.js

```diff
@@ -1,3 +1,208 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('tbody');
+const startButton = document.querySelector('.start');
+const scoreGame = document.querySelector('.game-score');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('.message-start');
+
+const rows = 4;
+const cols = 4;
+let field;
+
+let score = 0;
+
+startButton.addEventListener('click', startGame);
+
+function setRundomNumber() {
+  const twoOrFour = Math.random() < 0.2 ? 4 : 2;
+
+  let found = false;
+
+  while (!found) {
```

Comment:
It is not recommended to use the `while (true)` loop. Instead, use a different looping method or use a boolean flag.